### ISSUE TITLE
Replace emoji with provided character

### ISSUE THREAD

**Issue by rahulupadhyay** [vdurmont/emoji-java#85]
THIS IS NOT AN ISSUE. BUT I DON"T KNOW HOW TO ADD LABEL. 

I really like this project it helps me a lot. 

I'm suppose to change emoji with particular character say "?". 

I've achieved it by creating method : 

```
/**
     * Replace all emojis with character
     *
     * @param str the string to process
     * @param chr the character to be replaced
     * @return the string with replaced character
     */
    public static String replaceAllEmojis(String str, final String chr) {
        EmojiParser.EmojiTransformer emojiTransformer = new EmojiParser.EmojiTransformer() {
            public String transform(EmojiParser.UnicodeCandidate unicodeCandidate) {
                return chr;
            }
        };

        return parseFromUnicode(str, emojiTransformer);
    }
```

Can you please add this function? I'm using it and its working fine. 

Thanks.

**Comment by cbedoy** [comment]
Why don't create a pull request ?

**Comment by vdurmont** [comment]
yeah a pull request would be fine!

**Comment by ewencluley-bw** [comment]
I see this change has been merged after the 4.0.0 release- could you perform another release to make this functionality available please. Thanks.
